Final_Project/Final_Project.vcxproj.filters

```diff
@@ -42,7 +42,11 @@
       <Filter>Header Files</Filter>
     </ClInclude>
     <ClInclude Include="Main.h">
+<<<<<<< HEAD
```

Comment:
there's a merge conflict, I had created similar directories in the Header directory for app, dl, and phys
We can keep those or just simply have all in one Header directory.

Reply:
Ah I totally missed that! I think that's from the temp `Main.h` file I had committed. I'll make it so we keep the separate directories in the Header directory.